src/containers/orders/cart/cart-item/cart-item.js

```diff
@@ -45,8 +48,10 @@ const CartItem = ({ item, setModalVisibility, setModalItem, cartOperations, prom
     getCartItem,
     changeSizeConstructor,
     getProductPriceWithPromoCode,
-    getProductPrice
+    getProductPrice,
+    removeFromCart
   } = cartOperations;
+  const { image, checkImage } = useProductImage();
 
   const onChangeQuantity = useCallback(
     _.debounce((value) => {
```

Comment:
We have useDebounce hook 
You can use it if you want)

Reply:
I haven't wanted but I used it)